lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
Tried switching to `Node.js v6.2.1` without applying my fix. The bug is still there. @Quramy When can we actually merge this? Is there anymore concerns?